### PR TITLE
Resolve #1302

### LINKED ISSUE #1302: v3.3.1 - Clarification as to meaning,
This requirement is currently as follows. (My emphasis). I had a question come up on this during a training course.

[3.3.1](https://github.com/OWASP/ASVS/blob/master/5.0/en/0x12-V3-Session-management.md#v33-session-termination)

> Verify that logout and expiration invalidate the session token, such that the back button or a downstream relying party does not resume an authenticated session, **including across relying parties**.

(The earliest version of the requirement I can find was added [here as 3.3.4](https://github.com/OWASP/ASVS/commit/64194cfc4e4b3229acb20e03812f626bc247ed43).)

My understanding of this requirement includes a few things. For the purposes of this point let's assume AzureAD is the identity provider (IdP) and Salesforce and ServiceNow are relying parties (RPs) where they use the user's AzureAD identity rather than managing authentication separately.

Understanding of the requirement:
* Logout from AzureAD invalidates the session identifier with AzureAD and the user will not be able to use AzureAD until they authenticate again.
* Pressing the back button will not result in seeing sensitive data from AzureAD or being able to resurrect the session with AzureAD.
* Browsing to Salesforce or ServiceNow should not resurrect the session with AzureAD.
* In fact, the last 4 words of the requirement ("_including across relying parties_") indicate that the user's session with Salesforce or ServiceNow should also be invalidated when the IdP session has been invalidated. After logging out in the IdP, the user should not be able to access these RPs without first reauthenticating to the IdP.
* However, if the user logs out from their specific session with one of the RPs (e.g. Salesforce), this requirement **does not** expect or require that this will also log the user out of other RPs or the IdP.

@vanderaj (or anyone else), do you understand this requirement differently to me?

### PR DESCRIPTION
**Pull request by tghosth**
This Pull Request relates to issue #1302

### CODE REVIEW

**Comment by tghosth** [comment]
@jmanico is this ok?

File: 5.0/en/0x12-V3-Session-management.md [hunk 1]
```diff
@@ -37,7 +37,7 @@ L1 in this context is IAL1/AAL1, L2 is IAL2/AAL3, L3 is IAL3/AAL3. For IAL2/AAL2
 
 | # | Description | L1 | L2 | L3 | CWE | [NIST &sect;](https://pages.nist.gov/800-63-3/sp800-63b.html) |
 | :---: | :--- | :---: | :---: | :---: | :---: | :---: |
-| **3.3.1** | Verify that logout and expiration invalidate the session token, such that the back button or a downstream relying party does not resume an authenticated session, including across relying parties. ([C6](https://owasp.org/www-project-proactive-controls/#div-numbering)) | ✓ | ✓ | ✓ | 613 | 7.1 |
+| **3.3.1** | Verify that logout and expiration invalidate the session token, such that the back button or a downstream relying party cannot resume an authenticated session. ([C6](https://owasp.org/www-project-proactive-controls/#div-numbering)) | ✓ | ✓ | ✓ | 613 | 7.1 |
```
Comment (elarlang) [inline 5.0/en/0x12-V3-Session-management.md:40]: `[MODIFIED]`

Comment (tghosth) [reply 5.0/en/0x12-V3-Session-management.md]: Fixed that @elarlang

**Review by elarlang — APPROVED** [review]

**Comment by tghosth** [comment]
Gonna give @jmanico a week to review this and then ship it :)

**Comment by danielcuthbert** [comment]
Had a look and support the changes @tghosth so approving this and getting it merged

**Pull request merged**